Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
     }
 
     triggers {
-        upstream(upstreamProjects: "Codewind/codewind-installer/${env.BRANCH_NAME}", threshold: hudson.model.Result.SUCCESS)
+        upstream(upstreamProjects: "Codewind/codewind-installer/${env.BRANCH_NAME},Codewind/codewind-eclipse/${env.BRANCH_NAME}", threshold: hudson.model.Result.SUCCESS)
```

Comment:
When the installer is built, it cause both an Eclipse build and an IntelliJ build to be kicked off.  Then when the Eclipse build finishes, it will cause an additional IntelliJ build to be kicked off.

Would it make sense to have the IntelliJ build only depend on the Eclipse build, so that we don't get an additional useless build?

Reply:
Makes sense - I removed 'codewind-installer' from upstream project.